### DISCUSSION TITLE
Tags in Search Filter Stopped working after upgrade to 4.7

### DISCUSSION THREAD

**Discussion by nthoman** [microsoft-search/pnp-modern-search#2409] [General]
I went from 4.3 to 4.7 and now my "PnP - Search Filters" stopped working for "Tags" only. My other search filters work fine. The search filter will show the correct number of records but when I try to filter it down the Search Results will change to 0 records.

![image](https://user-images.githubusercontent.com/48287847/193829915-2848883d-9b4b-4e69-90b4-4cfb22406e06.png)
 
![image](https://user-images.githubusercontent.com/48287847/193830099-8df16899-72f8-4f8c-b79c-18f154a883cd.png)

![image](https://user-images.githubusercontent.com/48287847/193830391-fa8c2289-440b-4515-8b5c-e12405d01afe.png)

**Comment by nthoman** [comment]
Rolling back to 4.3 and the search behaves as expected.

![image](https://user-images.githubusercontent.com/48287847/193832916-142f43f1-c261-4a0e-8cab-fbfdd64ea2be.png)

**Reply by linahin** [reply]
Hello @nthoman how do you get to display only the term label and not the entire path ? mine is showing the entire path

**Reply by wobba** [reply]
> Hello @nthoman how do you get to display only the term label and not the entire path ? mine is showing the entire path

Full path is due to using ows_column name which contains the setting in the library, which for your case is show full term path in SharePoint. Take a look at "Enable localization" at https://microsoft-search.github.io/pnp-modern-search/usage/search-results/data-sources/sharepoint-search/

**Comment by wobba** [comment]
@nthoman can you retest with the latest version or dev branch?
